### PR TITLE
[`Frontend`]: Fix avatar

### PR DESCRIPTION
**Pull request by tokitouq**
- [x] https://github.com/baseplate-admin/CoreProject/pull/3163#discussion_r1428878459
- [x] https://github.com/baseplate-admin/CoreProject/pull/3163#discussion_r1428879203

### CODE REVIEW

File: backend/django_core/templates/anime/_layout.html [hunk 1]
```diff
@@ -35,14 +35,24 @@
...
+                      <ul tabindex="0" class="dropdown-content md:top-[4vw] md:text-[0.9vw] md:p-[0.5vw] bg-base-100 menu md:rounded-[0.75vw]">
+                        <div class="md:p-[0.75vw] md:py-[0.5vw] flex flex-col leading-none md:gap-[0.35vw]">
+                            <span class="md:text-[1vw] font-bold">Soraamamiya#0001</span>
+                            <span class="md:text-[0.75vw]">sora@coreproject.moe</span>
+                        </div>
+                        <div class="divider md:mb-[0.1vw] mt-0"></div>
+                        <li><a class="md:p-[0.75vw] leading-none md:rounded-[0.5vw] hover:bg-primary hover:text-white">Profile</a></li>
+                        <li><a class="md:p-[0.75vw] leading-none md:rounded-[0.5vw] hover:bg-primary hover:text-white">My List</a></li>
+                        <li><a class="md:p-[0.75vw] leading-none md:rounded-[0.5vw] hover:bg-primary hover:text-white">Theme</a></li>
+                        <li><a class="md:p-[0.75vw] leading-none md:rounded-[0.5vw] hover:bg-primary hover:text-white">Settings</a></li>
```
Comment (tokitouq) [inline backend/django_core/templates/anime/_layout.html]: Add icons here

File: backend/django_core/templates/anime/_layout.html [hunk 2]
```diff
@@ -35,14 +35,24 @@
...
-                    <button class="avatar" aria-label="Avatar" aria-expanded="false">
-                        <figure
-                            class="avatar flex aspect-square text-surface-50 font-semibold justify-center items-center overflow-hidden isolate bg-surface-400-500-token w-12 md:w-[3.125vw] rounded-[0.4rem] md:rounded-[0.375vw]"
-                            data-testid="avatar"
-                        >
-                            <img class="avatar-image w-full h-full object-cover" style="" src="/images/Avatar.avif" alt="" />
-                        </figure>
-                    </button>
+                    <div class="dropdown dropdown-end flex">
+                        <div tabindex="0" role="button" class="avatar">
```
Comment (tokitouq) [inline backend/django_core/templates/anime/_layout.html:39]: conditional rendering here for `request.user.is_authenticated`
show `log in/ sign up` button

File: backend/django_core/templates/anime/_layout.html [hunk 3]
```diff
@@ -35,14 +35,48 @@
...
+                            <div class="md:w-[3vw] md:rounded-[0.5vw]">
+                                <img src="https://daisyui.com/images/stock/photo-1534528741775-53994a69daeb.jpg" />
+                            </div>
+                        </div>
+                        <ul tabindex="0" class="dropdown-content md:top-[4vw] md:text-[0.9vw] md:p-[0.5vw] bg-base-100 menu md:rounded-[0.75vw]">
+                            <div class="md:p-[0.75vw] md:py-[0.5vw] flex flex-col leading-none md:gap-[0.35vw]">
+                                <span class="md:text-[1vw] font-bold">Soraamamiya#0001</span>
```
Comment (baseplate-admin) [inline backend/django_core/templates/anime/_layout.html:46]: Yessss

Comment (tokitouq) [reply backend/django_core/templates/anime/_layout.html]: My Pleasure Sir :heart:

**Comment by tokitouq** [comment]
@baseplate-admin  **Final**
Screenshot from 2023-12-17 09-47-02

**Pull request merged**